templates/support_channels_fi.html

```diff
@@ -0,0 +1,111 @@
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener" href="https://apluslms.github.io/">{% trans "Aplus manual" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Community" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://teams.microsoft.com/l/team/19%3a147685e6a8fc409295188befeec77113%40thread.skype/conversations?groupId=b78ab778-8838-4180-940d-b9183ddc3a7c&tenantId=ae1a7724-4041-4462-a6dc-538cb199707e">
+						{% trans "Microsoft Teams" %}
+					</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://github.com/apluslms/a-plus/issues">{% trans "Report an issue in Github" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "News" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://wiki.aalto.fi/display/EDIT/EDIT#EDIT-News">{% trans "Announcements" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://status.cs.aalto.fi/d/r0N6S9jik/main?orgId=1">{% trans "Status" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://github.com/orgs/apluslms/projects/2#card-50639420">{% trans "Current Development" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% blocktrans with brand=brand %}Other servers that use {{brand}}{% endblocktrans %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener" href="https://wiki.aalto.fi/display/EDIT/Exam+studio+exam+with+Aplus+exercises">{% trans "Exam server" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener" href="https://minus.cs.aalto.fi/">{% trans "Testing environment" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Learning" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://wiki.aalto.fi/display/EDIT/Accessible+Teaching">{% trans "Accessible Teaching" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://wiki.aalto.fi/display/EDIT/Exam+studio+exam+with+Aplus+exercises">
+						{% blocktrans with brand=brand %}Exam studio with {{brand}}{% endblocktrans %}
+					</a>
+				</p>
+			</div>
+		</div>
+	</div>
+</section>
+
+<section id="contact-us" class="bg-gray text-center">
+	<div>
+		<img alt="decorative icon" class="img-responsive center-block" src="{% static 'images/search-clipart.png' %}" />
+	</div>
+	<div>
+		<h3>{% trans "Can't find what you're looking for?" %}</h3>
+	</div>
+	<div class="btn-group" role="group" aria-label="...">
+		<a target="_blank" rel="noopener" class="btn btn-primary"
```

Comment:
You have used colourful buttons just for decorating. Our new button styles from the summer 2020 forbid colourful buttons. Only the semantic colours can be used. 

I would raise the course instance request from the bottom to one of the cards. If we have the support email address in the bottom as a button, the button can use the a-plus primary colour. Though, I think the email address could also be more visible. Now the address itself is not rendered in the page at all.